compiler/qsc_frontend/src/resolve.rs

```diff
@@ -756,15 +833,23 @@ impl GlobalTable {
         for (name, res) in builtins {
             core.insert(name, res);
         }
-        let mut tys: FxHashMap<Rc<str>, FxHashMap<Rc<str>, Res>> = FxHashMap::default();
-        tys.insert("Microsoft.Quantum.Core".into(), core);
+
+        let mut scope = GlobalScope::default();
+        let ns = scope.insert_or_find_namespace(vec![
```

Comment:
It could be helpful if you could centralize the actual name of these prelude namespaces near, or in the prelude structure. That way if we ever want to change the name of the namespace (like remove the "Microsoft.Quantum" part), we'd only need to change it in the centralized location.